Userland/Services/WebDriver/Client.cpp

```diff
@@ -442,6 +443,23 @@ ErrorOr<JsonValue, HttpError> Client::handle_get_timeouts(Vector<AK::StringView>
     return make_json_value(timeouts);
 }
 
+// 9.2 Set Timeouts, https://w3c.github.io/webdriver/#dfn-set-timeouts
+// POST /session/{session id}/timeouts
+ErrorOr<JsonValue, HttpError> Client::handle_set_timeouts(Vector<AK::StringView> const& parameters, AK::JsonValue const& payload)
```

Comment:
Is there a special reason the contents of this live here? All other handlers forward to their counterparts in `Session`.

Reply:
Eh, I didn't really see the point but sure :)